frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/VerifyCodeHandler.java

```diff
@@ -374,4 +381,17 @@ private JourneyType getJourneyType(
         }
         return journeyType;
     }
+
+    private String MFACode(APIGatewayProxyRequestEvent input) {
```

Comment:
`VerifyCodeHandler extends BaseFrontendHandler<VerifyCodeRequest>` which means that you already have all of the JSON parsing as a 'freebie'.

The MFA code entered is available therefore via  `codeRequest.getCode()`. You don't need a whole new function to try and wrangle JSON directly